vyper/builtins/functions.py

```diff
@@ -1061,6 +1061,26 @@ def fetch_call_return(self, node):
 
         kwargz = {i.arg: i.value for i in node.keywords}
 
+        delegate_call = kwargz.get("is_delegate_call", False)
```

Comment:
`False` is not of type VyperNode, so i'm not sure you're comparing what you expect here (at this stage in compilation, default values are not evaluated yet)

Reply:
for instance, `raw_call(..., delegate_call=False, static_call=True)` will raise an ArgumentException

Reply:
which is maybe a good thing! using both keywords seems like a mistake.

Reply:
maybe use `None` though